packages/LeptonSelector/LeptonSelector.C

```diff
@@ -673,14 +749,23 @@ Bool_t LeptonSelector::isLooseLepton(Lepton lep){
     if(!getGoodVertex(iTight)) return false;
     return true;
   }
-  if(gSelection == iWZSelec){ // Todo lo que esta mas para alla se hace dentro del paquete de analisis
+  else if(gSelection == iWZSelec){ // Loose leptons are common to all tight WP
     if(lep.isMuon){
-      if(lep.p.Pt() < 10) return false;
+      if(lep.p.Pt() < 5) return false;
       if(TMath::Abs(lep.p.Eta()) > 2.4) return false;
+			if(!getGoodVertex(iLoose)) return false;
+			if(!getSIPcut(8)) return false;
+			if(!getminiRelIso(iLoose)) return false;
     }
     if(lep.isElec){
-      if(lep.p.Pt() < 10) return false;
+      if(lep.p.Pt() < 7) return false;
```

Comment:
7?

Reply:
Loose MVAElectrons as defined for the EWKino analysis (and its inheritors I guessed...) have this p_T cut. I thought it might be useful to have the ID as synchronized as possible for the future (if we ever have non-p_T skimmed trees)

Reply:
Good!